test/http_client.c

```diff
@@ -409,6 +408,9 @@ usage (const char *prog)
 "Usage: %s [opts]\n"
 "\n"
 "Options:\n"
+"	-6 IPv6		The client will try to connect via IPv6\n"
```

Comment:
This flag should be moved toward the end of the list.  Also, please indent it similar to others.